### DISCUSSION TITLE
Error when creating *.apk with AIR 33

### DISCUSSION THREAD

**Discussion by pcrivatz** [airsdk/Adobe-Runtime-Support#635] [Q&A]
![AdobeError](https://user-images.githubusercontent.com/40854507/62925710-9eaa3680-bdbb-11e9-9a31-ea06541e4d94.png)

Hello. When I am trying to create a build for android with AIR 33.0.1.220, it gives me the error in the screen shot. I am using milkman's *.ane for AdMob. When I use AIR 32 it works just fine. 
My main question is, do *.ane files need to be also created using AIR 33 and to have support for 64-bit?

**Comment by ajwfrost** [comment]
Hi

Yes, ANEs need to be updated with support for 64-bit targets as well. Sadly I think Milkman are not currently supporting their old ANEs which means you may need to find an alternative option here.

thanks

**Comment by pcrivatz** [comment]
Thank you very much :)

**Comment by jeuxdaily** [comment]
please link AIR SDK 33
